### PR TITLE
ci: split ovn/ovs into base image

### PR DESCRIPTION
**Pull request by oilbeater**
#### What type of this PR
Examples of user facing changes:
- CI

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -42,15 +40,23 @@ build-dev-images: build-bin
 build-dpdk:
 	docker buildx build --cache-from "type=local,src=/tmp/.buildx-cache" --cache-to "type=local,dest=/tmp/.buildx-cache" --platform linux/amd64 -t $(REGISTRY)/kube-ovn-dpdk:19.11-$(RELEASE_TAG) -o type=docker -f dist/images/Dockerfile.dpdk1911 dist/images/
 
+.PHONY: build-base-amd64
```
Comment (zhangzujian) [inline Makefile:43]: should be `base-amd64`

File: Makefile [hunk 2]
```diff
@@ -42,15 +40,23 @@ build-dev-images: build-bin
...
+base-amd64:
+	docker buildx build --cache-from "type=local,src=/tmp/.buildx-cache" --cache-to "type=local,dest=/tmp/.buildx-cache" --platform linux/amd64 --build-arg ARCH=amd64 -t $(REGISTRY)/kube-ovn-base:$(RELEASE_TAG)-amd64 -o type=docker -f dist/images/Dockerfile.base dist/images/
+
+.PHONY: build-base-arm64
```
Comment (zhangzujian) [inline Makefile:47]: should be `base-arm64`

File: dist/images/Dockerfile.base [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    python3 libmnl-dev libpcap-dev libtool libcap-ng-dev libssl-dev pkg-config \
+    python3-six libunbound-dev libunwind-dev dh-make fakeroot debhelper dh-python \
+    flake8 python3-sphinx graphviz groff -y
+
+RUN cd /usr/src/ && \
+    git clone -b branch-2.15 --depth=1 https://github.com/openvswitch/ovs.git && \
+    cd ovs && \
+    curl https://github.com/kubeovn/ovs/commit/df1c802f568be2af84aa81372fec46f5b09b4366.patch | git apply && \
+    curl https://github.com/kubeovn/ovs/commit/8dd4973bba021ce92eb9f42446d6c1c0e3474e25.patch | git apply && \
+
```
Comment (zhangzujian) [inline dist/images/Dockerfile.base:18]: Should we remove this empty line?

**Review by zhangzujian — CHANGES_REQUESTED** [review]
request changes

File: dist/images/Dockerfile.base (hunk 3 above)
Comment (oilbeater) [reply dist/images/Dockerfile.base]: all fixed

**Review by zhangzujian — APPROVED** [review]
LGTM

**Pull request merged**